lib/helper/zone-helper.ts

```diff
@@ -0,0 +1,147 @@
+/**
```

Comment:
Why is this separate file? Why not just fold it to the error handling in Zone.ts which already filters out some stack frames. We could just expand it to filter all?

I don't think most people would use this as it would require set up. I think having this as part of Zone.ts would make it so that everyone would benefit.